openlr/openlr_model.hpp

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include "geometry/latlon.hpp"
+#include "geometry/point2d.hpp"
+
+#include "std/limits.hpp"
+
+namespace openlr
+{
+// The form of way (FOW) describes the physical road type of a line.
+enum class FormOfAWay
```

Comment:
I've answered this already. This is the name of an object in OpenLR spec. I'd like to leave it as is, if you don't mind.